.chloggen/elasticsearchexporter_merge-geo-location.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
TODO to self: check if this change causes a mapping conflict if `lon` and `lat` are stored in separate fields in existing documents

Reply:
Confirmed there is no problem.